### DISCUSSION TITLE
Ammonia storage tank pool fire is possible ?

### DISCUSSION THREAD

**Discussion by Thiru606** [firemodels/fds#13859] [General]
hello Friends, Please help me whether ammonia storage tank "pool FIRE "is possible with FDS simulation or not can anybody help me in write input code

**Comment by drjfloyd** [comment]
It isn't clear what you are trying to do.  I suggest you look at the FDS documentation (Tech Guide, User's Guide, Validation Guide, and Verification Guide) and see if you find anything similar to what you are trying to do.

**Reply by Thiru606** [reply]
NH3 STORAGE TANK POOL FIRE IS POSSIBLE IF SO how to write react id for Combustion of Ammonia for FDS Simulation

**Reply by Thiru606** [reply]
ERROR(171): REAC, Not enough carbon for the CO_YIELD, SOOT_YIELD, and/or HCN_YIELD. (CHID: NH3_tank_fire)
forrtl: severe (157): Program Exception - access violation

**Reply by drjfloyd** [reply]
Please read the user's guide.  The error should be self explanatory. Ammonia is NH3.  It is not going to make CO, HCN, or carbonaceous soot. NH3 has no carbon.
